src/ert/analysis/_es_update.py

```diff
@@ -474,6 +478,14 @@ def adaptive_localization_progress_callback(
     num_obs = len(observation_values)
 
     smoother_snapshot.update_step_snapshots = update_snapshot
+    # Used as labels for observations in cross-correlation matrix.
+    # Say we have two observation groups "FOPR" and "WOPR" where "FOPR" has
```

Comment:
This is to accommodate having multiple "observed" values under the same observation key? I thought that every observation point needs to have a unique name already or not?

Reply:
Oh I see:
```
GENERAL_OBSERVATION MY_OBS_10 {
    DATA       = MY_RESPONSE;
    RESTART    = 10;
    OBS_FILE   = obs_10.txt;
};
```
Now I understand :+1: